eslint.config.mjs

```diff
@@ -0,0 +1,202 @@
+import { defineConfig } from "eslint/config";
+import reactPlugin from "eslint-plugin-react";
+import typescriptEslint from "@typescript-eslint/eslint-plugin";
+import globals from "globals";
+import tsParser from "@typescript-eslint/parser";
+import js from "@eslint/js";
+import * as eslintMdx from "eslint-mdx";
+import eslintConfigPrettier from "eslint-config-prettier/flat";
+import reactHooks from "eslint-plugin-react-hooks";
+import storybook from "eslint-plugin-storybook";
+
+export default defineConfig([
+    {
+        ignores: [
+            "**/node_modules/**",
+            "**/dist/**",
+            "**/.storybook/**",
+            "**/e2e/**",
+            "playwright.config.ts",
+        ],
+    },
+    js.configs.recommended,
+    reactPlugin.configs.flat.recommended,
```

Comment:
to also include the JSX runtime config recommended by `eslint-plugin-react` for React 17+

https://github.com/jsx-eslint/eslint-plugin-react?tab=readme-ov-file#flat-configs